src/session.ts

```diff
@@ -220,9 +220,13 @@ export class SessionManager implements Middleware {
         if (this.sessionSettings.integratedConsole.suppressStartupBanner) {
             this.editorServicesArgs += "-StartupBanner '' ";
         } else {
-            const startupBanner = `=====> ${this.HostName} Integrated Console v${this.HostVersion} <=====
+            const startupBanner = `${this.HostName} Extension v${this.HostVersion}
+Copyright (c) Microsoft Corporation.
+
+https://aka.ms/powershell
```

Comment:
@SydneyhSmith Is there a better link we could put here?

Reply:
This is what shows on startup? Seems fine to me...alternative would be a VSCode PowerShell link